src/options/imports/index_ops.js

```diff
@@ -0,0 +1,56 @@
+import index from 'src/search/search-index'
+
+const db = index.db._db
+
+async function downloadChunk(data, counter) {
+    chrome.downloads.download({
+        url: URL.createObjectURL(
+            new Blob([JSON.stringify(data)], { type: 'text/plain' }),
+        ),
+        filename: 'worldbrain_index_'.concat(counter, '.json'),
```

Comment:
Using string templates might be a nicer declarative way to handle string formatting. So something like `` `worldbrain_index_${counter}.json` `` here